### DISCUSSION TITLE
is this a primary article

### DISCUSSION THREAD

**Discussion by madelyn-redick** [Biol2299-Matthis-Fall2022/Biol2299-Matthis-Fall2022#35] [General]
is it primary
https://sci-hub.se/10.1016/0021-9290(91)90002-5

lmk what you think <3

**Comment by sahana-dhar** [comment]
Yes, I believe so because it states the purpose of this study, which sounds like it was ran by the researchers writing this article. They included the formulas and methods used to run their experiments as well.

**Reply by madelyn-redick** [reply]
thank you!

**Comment by jonmatthis** [comment]
man, look at those cool old school figures! 

Something like Figure 6 would've been seen as *crazy* high tech back when this came out
